chalice/cli/__init__.py

```diff
@@ -76,18 +76,22 @@ def cli(ctx, project_dir, debug=False):
 
 @cli.command()
 @click.option('--port', default=8000, type=click.INT)
+@click.option('--stage', default=DEFAULT_STAGE_NAME,
+              help=('Name of the Chalice stage to deploy to. '
```

Comment:
Could you update the docstring here to just say that this is the stage configuration we'll use for local mode?

Reply:
Sure, just finished changing it.  Good catch.